client/public.go

```diff
@@ -369,6 +369,44 @@ func (p *Profile) AddProfessionals(ctx context.Context, careTeamID string, proID
 	return nil
 }
 
+func (p *Profile) AddCareGiversToCareTeam(ctx context.Context, careTeamID string, cgIDs []string) error {
+	defer func() {
+		go clientTransport.CloseIdleConnections()
+	}()
+	conf := config.Current()
+	requestID := velacontext.GetContextRequestID(ctx)
+
+	url := fmt.Sprintf("%s/api/v1/admin/care-teams/%s/member", conf.Common.PublicBaseURI, careTeamID)
+	newMemberTmpl := `{"member":{"user_id": "%s", "owner_type": "Caregiver"}}`
+
+	for _, proID := range cgIDs {
+		jsonStr := fmt.Sprintf(newMemberTmpl, proID)
+
+		request, rerr := http.NewRequest("POST", url, bytes.NewBuffer([]byte(jsonStr)))
+		request.Header.Set("Content-Type", "application/json")
+		request.Header.Add("X-Vela-Request-Id", requestID)
+		request.Header.Set("Authorization", fmt.Sprintf("Bearer %s", p.AccessToken))
+		response, err := apiClient.Do(request)
+		if rerr != nil || err != nil || response == nil {
```

Comment:
This probably won't work the way you want.  If `rerr` is not `nil`, the `request.Header.Set/Add(...)` calls will fail, and cause a panic instead of a graceful recovery.  You need to check `rerr` immediately.

Reply:
check inserted